### DISCUSSION TITLE
HARD FAIL - delivery

### DISCUSSION THREAD

**Discussion by PankilBhatt** [postalserver/postal#1741] [Help with using Postal]
Hi,
I have a problem with delivery.

The Problem is with HARD FAIL

@gmail.com (550-5.7.1 [5.182.17.60 12] Our system has detected that this message is 550-5.7.1 likely unsolicited mail. To reduce the amount of spam sent to Gmail, 550-5.7.1 this message has been blocked.), 

@yahoo.com & @aol.com (553 5.7.2 [TSS09] All messages from IP will be permanently deferred; Retrying will NOT succeed. See https://postmaster.yahooinc.com/error-codes).

@hotmail – are delivery OK.

Thanks

**Comment by willpower232** [comment]
It sounds like the IP address or domain name you're sending from may have been added to a block list, you could verify with mxtoolbox.com or similar.

If it is not that then you would have to try contacting those companies to see if there is a resolution available to you.
